taskcluster/config.yml

```diff
@@ -142,6 +142,31 @@ release-promotion:
 scriptworker:
     scope-prefix: project:mozillavpn:releng
 
+mac-signing:
```

Comment:
I know we put this in the graph config in Gecko, but in your case it might be better to put into the kind, seeing as you only have one usage of it. (And honestly, in Gecko's case it probably belongs in a transform rather than the graph-level config.)

Totally up to; keeping it as is is fine too.

Reply:
Moving it into the kind configuration does sound cleaner. I'll give that a try and see what happens.